todolist.html

```diff
@@ -32,5 +32,22 @@
                 <i class='icon icon--left fa fa-trash-o'></i> Delete All
             </button>
         </div>
+        
+        <div>
+            <header>
+                Delete Task?
```

Comment:
You still need some kind of tag around 'Delete Task'

Reply:
Changed to h2 tag for now until styling can be added to modify the size properly